exasol_advanced_analytics_framework/resources/outputs/create_event_loop.sql

```diff
@@ -0,0 +1,564 @@
+CREATE OR REPLACE LUA SCRIPT "AAF_EVENT_LOOP"(json_str)  AS
+    table.insert(_G.package.searchers,
+        function (module_name)
+            local loader = package.preload[module_name]
+            if not loader then
+                error("Module " .. module_name .. " not found in package.preload.")
+            else
+                return loader
+            end
```

Comment:
The else case tends to add unnecessary extra noise especially in langauges where the blocks (syntax) itself produce significant noise (e.g. lua and ruby).

Imho it has a very bad signal to noise ratio here:
Signal:
* `return loader`
Noise:
* 1 additional block (branch)
* 1 additional keyword `else`

if you get rid of the additional branch, which will be executed in all non error cases it is way easier to read (imho)
```suggestion
           end
           return loader
```